three.js/src/threex/threex-artoolkitcontext.js

```diff
@@ -252,6 +252,7 @@ ARjs.Context.prototype.getProjectionMatrix = function (srcElement) {
     console.assert(this.arController, 'arController MUST be initialized to call this function')
     // get projectionMatrixArr from artoolkit
     var projectionMatrixArr = this.arController.getCameraMatrix();
+    console.log(projectionMatrixArr);
```

Comment:
printing the projectionMatrixArr and test artoolkit.min.js and artoolkit-nft.min.js respectively.
Test the dev exemple with the image (img.jpg) as source. Printing the values you will see that:

with the new artoolkit.nft.min.js 

`Float64Array(16) [1.9122345476570097, 0, 0, 0, 0, 2.5412900430297793, 0, 0, -0.015003890671989373, -0.004441884642872651, -1.0000002000000199, -1, 0, 0, -0.00020000002000000202, 0]
`
with the old artoolkit.min.js

`Float64Array(16) [1.9122345476570097, 0, 0, 0, 0, -2.5412900430297793, 0, 0, 0.015003890671989373, 0.004441884642872651, 1.0000002000000199, 1, 0, 0, -0.00020000002000000202, 0]`

It is clear the with the new lib the values are inverted so is not necessary to get the inverse of the matrix.